.github/workflows/build.yml

```diff
@@ -122,8 +124,12 @@ jobs:
     # Build components. Only components with out-of-date Turborepo hash will rebuilt, speeding up build time.
     - name: Build Components
       run: yarn build --cache-dir=".turbo"
+    # Unit Test components.
+    - name: Unit Test Components
+      run: yarn ci:test:${{ matrix.component-type}} --concurrency=10
     # Restore Storybook 'storybook-static'
     - name: Cache Storybook Static Directory
+      if: matrix.component-type == 'atoms' || matrix.component-type == 'molecules' || matrix.component-type == 'organisms' || matrix.component-type == 'pages'
```

Comment:
Shoild it also contain layouts?

Reply:
Not that I'm aware of, as we currently don't test these. However if and when we add tests, then yeah, this will needed to be added.